### PR TITLE
ToCode Solution: תרגול הרשאות

### PR DESCRIPTION
**Pull request by Jozefff**

### CODE REVIEW

File: 15_perms_lab/permslab.txt [hunk 1]
```diff
@@ -2,14 +2,30 @@
 
   - change permission of file src/main.c to r--r--r-- using chmod letter notation
 
+chmod u=r,g=r,o=r /home/yosef/wilcards/src/main.c
```
Comment (ynonp) [inline 15_perms_lab/permslab.txt:5]: Works. Can also use `chmod a=r ~/wildcards/src/main.c`

File: 15_perms_lab/permslab.txt [hunk 2]
```diff
@@ -2,14 +2,30 @@
...
+
   - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
 
+chmod 564 /home/yosef/wilcards/include/player.h
```
Comment (ynonp) [inline 15_perms_lab/permslab.txt:9]: 👍

File: 15_perms_lab/permslab.txt [hunk 3]
```diff
@@ -2,14 +2,30 @@
...
+
   - change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
 
+chmod 600 /home/yosef/wilcards/Doc/*
```
Comment (ynonp) [inline 15_perms_lab/permslab.txt:13]: Works. And there's a shorter version: `chmod 600 ~/wilcards/Doc/*`

File: 15_perms_lab/permslab.txt [hunk 4]
```diff
@@ -2,14 +2,30 @@
...
+
   - change permission of "lib" folder and all files and subdirectories inside it to rwxrwxrwx
 
+chmod -R 777  /home/yosef/wilcards/lib
```
Comment (ynonp) [inline 15_perms_lab/permslab.txt:17]: 👍

File: 15_perms_lab/permslab.txt [hunk 5]
```diff
@@ -2,14 +2,30 @@
...
+touch ~/test/secret
+chmod 000 ~/test/secret
+rm ~/test/secret
+rm -r test
+It's possible, for deleting file = editing this file, so if you can set permissions, which is editing, so you also can delete the file  and  directory = file, so the same.
 
+3. Read the output of "help umask". Explain in your own words what umask does, and how would you use it to set new files permissions to ------r--
 
+Sets new files' permissions or shows you their current value.
+umask 773
```
Comment (ynonp) [inline 15_perms_lab/permslab.txt:31]: 👍

**Review by ynonp — APPROVED** [review]
Awesome.

**Comment by Jozefff** [comment]
Hi Ynon,
:I have a question
If I want to set a job in crontab, could I use symbolic link to the file
with script or I need the real file?
Thanks

**Pull request closed without merging**